server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -410,6 +410,10 @@
     public final static ConfigKey<Long> IOPS_MAX_WRITE_LENGTH = new ConfigKey<Long>(Long.class, "vm.disk.iops.maximum.write.length", "Advanced", "0",
             "Maximum IOPS write burst duration (seconds). If '0' (zero) then does not check for maximum burst length.", true, ConfigKey.Scope.Global, null);
 
+    public static final ConfigKey<Boolean> ADD_HOST_ON_SERVICE_RESTART = new ConfigKey<Boolean>(Boolean.class, "add.host.on.service.restart", "Advanced", "true",
```

Comment:
If this is KVM specific can we add kvm in the global setting name @ravening ?

Reply:
@rhtyd i don't think this is KVM specific. The code path is same for all HV's I guess. Didn't test it on other Hv types.

Reply:
The agent only runs on KVM host, for the other two major hypervisors (VMware and XenServer) we don't run an agent on them.

Reply:
Didnt know that :P will add KVM in the name

Reply:
@rhtyd made the change